tests/Webapi/Webapi__Url__test.re

```diff
@@ -1,4 +1,6 @@
 open Webapi.Url;
 
 let params = URLSearchParams.make("key1=value1&key2=value2");
-URLSearchParams.forEach(Js.log2, params);
\ No newline at end of file
+URLSearchParams.forEach(Js.log2, params);
+let entries: array((string, string)) = Js.Array.from(URLSearchParams.entries(params));
```

Comment:
Could you remove the type annotation? In keeping with the othe variables in test files. Thanks.

Reply:
So for this one I get this error:

```
Error: The type of this expression, array('_weak1),
       contains type variables that cannot be generalized
```

When I look at the `js_array.ml` implementation in bucklescript, it's `'a array_like -> 'b array`, so the compiler doesn't like it.

Reply:
Ouch, well that's clearly wrong. Good opportunity for contributing a fix to bucklescript though!

Reply:
@illusionalsagacity ah, right, value restriction. What I'm doing nowadays is writing 'tester' functions like this:

```reason
let test_entries = params => params |> URLSearchParams.entries |> Js.Array.from;
```

Since it's inside a function, it shouldn't hit the value restriction.

Reply:
Btw the wrong signature is a known issue https://github.com/BuckleScript/bucklescript/issues/3272